frontend/src/utils/game/eventHandler.js

```diff
@@ -0,0 +1,55 @@
+import gameUtils from '@/utils/game/gameUtils';
+
+function eventHandler(canvas, scene, camera, renderer, controls) {
+  const map = scene.getObjectByName('map');
+  const ball = scene.getObjectByName('ball');
+  const redPaddle = scene.getObjectByName('redPaddle');
+  const bluePaddle = scene.getObjectByName('bluePaddle');
+
```

Comment:
캔버스를 클릭하기 전에는 keydown 이벤트를 인식 못하는 것 같습니다!
```suggestion
  canvas.focus();
```
이렇게 포커스를 해줘야하는 것 같아요